### ISSUE TITLE
Hidden text elements are read by screen readers

### ISSUE THREAD

**Issue by AleksueiR** [fgpv-vpgf/fgpv-vpgf#1326]
Elements hidden by `visibility: hidden` are not supposed to be read by screen readers: http://webaim.org/techniques/css/invisiblecontent/

The reality is different, however:
- JAWS: doesn't read
- NVDA: reads on mouse over only :( (https://github.com/nvaccess/nvda/issues/5320)
- test more screen readers ...

In the full-screen mode NVDA will read host page content when moused over. Hiding such content with other means (display: none, text-indent, position: absolute) is complicated as it's hard to restore true previous values (CSS [revert](https://developer.mozilla.org/en-US/docs/Web/CSS/revert) value is not widely supported).

**Comment by AleksueiR** [comment]
The NVDA issue is assigned low priority, and since they have > 2000 open issues, it's unlikely it will be fixed in the near future.
